tests/monitoring/vm_monitoring.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 	Context("VMI metrics", func() {
 		It("should have kubevirt_vmi_phase_transition_time_seconds buckets correctly configured", func() {
 			vmi := libvmifact.NewGuestless()
-			tests.RunVMIAndExpectLaunch(vmi, 240)
+			libvmops.RunVMIAndExpectLaunch(vmi, 240)
```

Comment:
This change was suppose to be part of the 1st commit, right?

Reply:
Right, ack